v2/samples/dataprotection/v1api20231101/v1api20231101_backupvaultsbackupinstance.yaml

```diff
@@ -0,0 +1,55 @@
+# Samples in this directory are shared which are required to be deployed first as Prerequisites before deploying this sample backup instance yaml.
+# Prerequisites: https://learn.microsoft.com/azure/backup/azure-kubernetes-service-cluster-backup-concept
+# Permissions needed: https://learn.microsoft.com/azure/backup/azure-kubernetes-service-cluster-backup-concept#required-roles-and-permissions
+apiVersion: dataprotection.azure.com/v1api20231101
+kind: BackupVaultsBackupInstance
+metadata:
+  name: aso-sample-backupinstance
+  namespace: default
+spec:
+  owner:
+    name: aso-sample-backupvault
+  properties:
+    objectType: BackupInstance
+    friendlyName: aso-sample-backupinstance
+    dataSourceInfo:
+      objectType: Datasource
+      datasourceType: Microsoft.ContainerService/managedClusters
+      resourceName: aso-sample-aks
+      resourceLocation: westus2
+      resourceType: Microsoft.ContainerService/managedClusters
+      resourceUri: /subscriptions/f0c630e0-2995-4853-b056-0b3c09cb673f/resourceGroups/aso-sample-rg/providers/Microsoft.ContainerService/managedClusters/aso-sample-aks
```

Comment:
replace the guid with `00000000-0000-0000-0000-000000000000`

Reply:
Result should be `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/aso-sample-rg/providers/Microsoft.ContainerService/managedClusters/aso-sample-aks`

Reply:
changed this